app/src/main/java/com/pajato/argus/ListAdapter.kt

```diff
@@ -14,20 +15,38 @@ class ListAdapter(val items: MutableList<Video>) : RecyclerView.Adapter<ListAdap
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        val titleTextView: TextView = holder.layout.findViewWithTag("title")
+        // Fill in the data and set an onClick for the delete button
+        val titleTextView = holder.layout.findViewWithTag<TextView>("title")
```

Comment:
Let's talk about this in the pair session.  Seems to me an id is just fine and using a tag is not necessary.